Добавлены стили для вьюпортов 320+ и 768+

mintybubble opened the pull request.

mintybubble committed:
Добавлены стили для вьюпортов 320+ и 768+

Sunwart commented on src/sass/layout/_contacting.scss:
```diff
@@ -0,0 +1,239 @@
+  }
+}
+
+.contact-page__contacts-item-link {
```
не задано font-size для класу

Sunwart commented on src/sass/layout/_contacting.scss:
```diff
@@ -0,0 +1,239 @@
+    margin-bottom: 68px;
+  }
+}
+.contact-page__worktime-item:nth-child(2) {
```
не задано font-size для класу

Sunwart reviewed (changes requested):
- на класи contact-page__contacts-item-link та contact-page__worktime-item треба задати font-size
- <ul class="list">
            <li class="contact-page__worktime-item">Пн-Пт:</li>
            <li class="contact-page__worktime-item">З 8:00 До 19:00</li>
            <li class="contact-page__worktime-item">Сб, Нд:</li>
            <li class="contact-page__worktime-item">З 10:00 До 17:00</li>
          </ul>
          
          **краще:** 
          <ul class="list">
            <li> 
            <p class="contact-page__worktime-item">Пн-Пт:</p>
            <p class="contact-page__worktime-item">З 8:00 До 19:00</p>
            </li>
            <li>
            <p class="contact-page__worktime-item">Сб, Нд:</p>
            <p class="contact-page__worktime-item">З 10:00 До 17:00</p>
             </li>
          </ul>

- Текст *Прикріпіть зображення... зараз і на мобільній і на планшетній версії переноситься не так, як на макеті

- і найголовніше - що відбувається з формою, коли вона заповнена тестовими даними - скріншот
![1](https://user-images.githubusercontent.com/83410878/129493862-69603f59-fcb7-49a0-9438-f566d1a17fa7.PNG)

А в іншому - круто! :)

mintybubble committed:
fixes

пофиксила все, кроме *Прикріпіть зображення... , пока не придумала как это сделать

Sunwart reviewed (approved):
Дякую!
